src/storybook/stand-alone-documentaion/typography/examples-table/example-table.module.scss

```diff
@@ -0,0 +1,23 @@
+.table {
+  $border: 1px solid var(--layout-border-color);
```

Comment:
ideally, each token we use in the stories should be with `sb-` prefix - coming from vibe-storybook-components